### DISCUSSION TITLE
github desktop fails to clone repositor: filename too long

### DISCUSSION THREAD

**Discussion by alex-s168** [MartinTheDragon/Nuclear-Tech-Mod-Remake#35] [General]
when i try to clone the repo i get this errors:

error: unable to create file 
src/datagen/generated/resources/data/nucleartech/advancements/recipes/nucleartech_parts/cobalt_powder_and_iron_powder_and_copper_powder_and_tiny_lithium_powder_from_centrifuging_crystals_from_centrifuge.json: Filename too long
error: unable to create file src/datagen/generated/resources/data/nucleartech/advancements/recipes/nucleartech_parts/plutonium_powder_and_cobalt_powder_and_niobium_powder_and_nitanium_mix_from_centrifuging_crystals_from_centrifuge.json: Filename too long

**Comment by MartinTheDragon** [comment]
Hi there!
Maybe try out this solution? <https://stackoverflow.com/a/51579051>
Either way, the generated naming of these files is a bit stupid, I could remove the `from_centrifuge` part...

**Reply by alex-s168** [reply]
it works now 

thanks!
